scripts/eval_attribution.py

```
@@ -0,0 +1,164 @@
+    return positions
+
+
+def white_to_move(position_command: str) -> bool:
+    if " startpos" in position_command:
+        moves = position_command.partition(" moves ")[2].split()
+        return len(moves) % 2 == 0
+    fen = position_command.partition(" fen ")[2].partition(" moves ")[0].split()
+    return len(fen) >= 2 and fen[1] == "w"
+
+
+def rival_raw(engine: UciEngine, use_nnue: bool) -> int:
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)